.circleci/unittest/windows/scripts/install.sh

```diff
@@ -26,13 +26,27 @@ else
 fi
 
 printf "Installing PyTorch with %s\n" "${cudatoolkit}"
-conda install -y -c "pytorch-${UPLOAD_CHANNEL}" "pytorch-${UPLOAD_CHANNEL}"::pytorch "${cudatoolkit}" pytest
+conda install -y -c "pytorch-${UPLOAD_CHANNEL}" -c conda-forge "pytorch-${UPLOAD_CHANNEL}"::pytorch "${cudatoolkit}" pytest
```

Comment:
Do you remember why adding conda-forge was needed here?

Reply:
Because the nightly package could be installed without conda-forge
https://github.com/pytorch/vision/pull/4058#issuecomment-866662203